lib/OpenQA/Worker/Job.pm

```diff
@@ -439,9 +441,8 @@ sub _stop_step_5_upload {
         }
 
         # upload other logs and files
-        for my $file (
-            qw(video.ogv video_time.vtt vars.json serial0 autoinst-log.txt serial_terminal.txt virtio_console.log worker-log.txt)
-          )
+        my @files_to_upload = split(',', $self->info->{settings}->{AUTOINST_UPLOAD_FROM_POOL} // '') // UPLOAD_FROM_POOL;
```

Comment:
Suggest:
```suggestion
        my @files_to_upload = split(',', $self->info->{settings}->{AUTOINST_UPLOAD_FROM_POOL} // '') || UPLOAD_FROM_POOL;
```

Reply:
Another thing which is "os-autoinst" related are these rename things like: https://github.com/os-autoinst/openQA/pull/2269/files#diff-2559347dfdfd46e3d678b4f8823f2d7fR452
Do we need to make this somehow also configurable?

Reply:
I don't know why the replacement is done here and not already done correctly in os-autoinst. I don't see a compelling reason to not propose a change in os-autoinst. But making it configurable might be beneficial as well.

Reply:
> Another thing which is "os-autoinst" related are these rename things like: https://github.com/os-autoinst/openQA/pull/2269/files#diff-2559347dfdfd46e3d678b4f8823f2d7fR452
> Do we need to make this somehow also configurable?


I personally don't see sense in such replacement , so think this line should be deleted . But I think it worth a separate PR and separate discussion as there might be people who will be against "renaming" of serial_terminal.txt . Or maybe actual fix for line which you mention is rename virtio_console.log into serial_terminal.log ?

Reply:
Providing files as .txt will probably provide a hint to browsers that the mime type is different so that the browser knows, based on user preferences, how to handle simple text files, e.g. show them directly in the browser instead of downloading as for binary files. However that part never worked properly for me, either I am always offered a download or the browser tries to open binary files directly in browser with garbled text.

Reply:
I think the reason for the replacement is, that a "common" user get the same output names, unrelated on specific background implementations.

To make it configurable, I would just allow a syntax like `realname=>destname` in `AUTOINST_UPLOAD_FROM_POOL` and use it if present.

Reply:
> Providing files as .txt will probably provide a hint to browsers that the mime type is different so that the browser knows, based on user preferences, how to handle simple text files, e.g. show them directly in the browser instead of downloading as for binary files. However that part never worked properly for me, either I am always offered a download or the browser tries to open binary files directly in browser with garbled text.

I am not against ".txt" ending of logs I am against to have them different in pool directory and as resulting asset. Do you see any issue to name  `virtio_console.log` as `serial_terminal.txt` initially to not bother with renaming at all ?

Reply:
no, looks also fine for me.